docs/docs/cli-commands.md

```diff
@@ -1903,6 +1904,38 @@ In order to use [Netlify Dev](https://www.netlify.com/products/dev/) you need to
 
 > Note: To detect the RedwoodJS framework, please use netlify-cli v3.34.0 or greater.
 
+### setup package
+
+This command takes a published npm package that you specify, performs some compatibility checks, and then executes it's bin script. This allows you to use third-party packages that can provide you with an easy-to-use setup command for the particular functionality they provide.
+
+This command behaves similarly to `yarn dlx` but will attempt to confirm compatibility between the package you are attempting to run and the current version of Redwood you are running. You can bypass this check by passing the `--force` flag if you feel you understand any potential compatibility issues.
+
+```
+yarn redwood setup package <npm-package>
+```
+
+| Arguments & Options | Description              |
+| :------------------ | :----------------------- |
+| `--force, -f`       | Overwrite existing files |
+
+**Usage**
+
+Run the made up `@bluewoodjs/rw-setup` package:
+```bash
+~/redwood-app$ yarn rw setup package @bluewoodjs/rw-setup
+```
+
+Run the same package but using a particular npm tag and avoiding any compatibility checks:
+```bash
+~/redwood-app$ yarn rw setup package @bluewoodjs/rw-setup@beta --force
+```
+
+**Compatibility Checks**
+
+We perform a simple compatibility check in an attempt to make you aware of potential compatibility issues with setup packages you might wish to run. This works by examining the version of `@redwoodjs/core` you are using within your root `package.json`. We compare this value with a compatibility range the npm package specifies in the `engines.redwoodjs` field of its own `package.json`. If the version of `@redwoodjs/core` you are using falls outside of the compatibility range specified by the package you are attempting to run, we will warn you and ask you to confirm you wish to continue.
+
+It's the responsibility of the authors of those npm packages to specify the correct range of compatibility so you should always research the packages you use with this command. Especially since they will be executing code on your machine!
```

Comment:
```suggestion
It's the author of the npm package's responsibility to specify the correct compatibility range, so **you should always research the packages you use with this command**. Especially since they will be executing code on your machine!
```